apps/api/src/caching/helpers.ts

```diff
@@ -30,20 +33,24 @@ export const Memoize = (options?: MemoizeOptions) => (target: object, propertyNa
 export async function cacheResponse<T>(seconds: number, key: string, refreshRequest: () => Promise<T>, keepData?: boolean): Promise<T> {
   const duration = seconds * 1000;
   const cachedObject = cacheEngine.getFromCache(key) as CachedObject<T> | undefined;
-  // console.log(`Cache key: ${key}`);
+  // logger.info(`Cache key: ${key}`);
```

Comment:
why commented? perhaps not needed? You may also use `logger.debug` and uncomment if you want to avoid it in some envs

Reply:
Yeah I think that was to quickly uncomment when debugging. I'll switch to using `logger.debug`.